global.json

```diff
@@ -3,6 +3,7 @@
     "dotnet": "8.0.100-alpha.1.23061.8"
   },
   "msbuild-sdks": {
+    "Microsoft.Build.NoTargets": "3.5.0",
```

Comment:
nit: it's generally a good idea to use the latest version for SDK. Please update to [3.7.0](https://www.nuget.org/packages/Microsoft.Build.NoTargets/3.7.0)